### PR TITLE
Support proper Websocket host and port settings in root API endpoint

### PR DESCRIPTION
**Pull request by krish7919**
Kubernetes deployment changes as per #1703 and #1626.

### CODE REVIEW

File: docs/server/source/production-deployment-template/node-config-map-and-secrets.rst [hunk 1]
```diff
@@ -123,6 +123,13 @@ receive requests from NGINX instance.
 It is set to ``80`` by default.
 
 
+vars.bigchaindb-wsserver-advertised-scheme
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``bigchaindb-wsserver-advertised-scheme`` is the protocol used to access
+the WebSocket API in BigchainDB. This can be set to 'wss' (default) or 'ws'.
```
Comment (ttmc) [inline docs/server/source/production-deployment-template/node-config-map-and-secrets.rst]: The default is ws, no? In the BigchainDB config file (JSON), you'd put it in double quotes, e.g. "ws" or "wss"

Comment (ttmc) [inline docs/server/source/production-deployment-template/node-config-map-and-secrets.rst]: Oh, regarding default, I guess you're setting the default to "wss" in the `config-map.yaml` file

Comment (krish7919) [inline docs/server/source/production-deployment-template/node-config-map-and-secrets.rst]: @ttmc The default is what we put in in the `config-map.yaml`.

Comment (krish7919) [inline docs/server/source/production-deployment-template/node-config-map-and-secrets.rst]: :+1: 
Better yet, I will rephrase that as per the existing documentation of default values.

**Comment by krish7919** [comment]
FYI: Output of the multi-node test cluster:
```
{
  "api": {
    "v1": {
      "assets": "/api/v1/assets/",
      "docs": "https://docs.bigchaindb.com/projects/server/en/v1.1.0.dev/http-client-server-api.html",
      "outputs": "/api/v1/outputs/",
      "statuses": "/api/v1/statuses/",
      "streams": "wss://main.ipdb.io:443/api/v1/streams/valid_transactions",
      "transactions": "/api/v1/transactions/"
    }
  },
  "docs": "https://docs.bigchaindb.com/projects/server/en/v1.1.0.dev/",
  "keyring": [
    "3waU13ezFiM2vow1j2WkzsLQax4ERrLUXF5oeMj685pL",
    "7bCviPDs7cmvDZM3G2UCFvAiraZEw4LJixvbZsdXx3c"
  ],
  "public_key": "Bf9dV8AcD5r5XhMTYrBUuurJ8cXJostxpgMUrfNAEh6g",
  "software": "BigchainDB",
  "version": "1.1.0.dev"
}
```

**Pull request merged**